homeassistant/components/hive/icons.json

```diff
@@ -6,9 +6,14 @@
       },
       "hot_water": {
         "default": "mdi:hand-water"
+      }
+    },
+    "sensor": {
+      "heating": {
+        "default": "mdi:radiator"
       },
-      "temperature": {
-        "default": "mdi:thermometer"
```

Comment:
Why do we remove the binary sensor one?

Reply:
because that translation key is not being used in the binary sensor (and it was me that added it recently)